astropy/utils/compat/misc.py

```diff
@@ -46,9 +46,11 @@ def override__dir__(f):
     Example
     -------
 
-    @override__dir__
-    def __dir__(self):
-        return ['special_method1', 'special_method2']
+    Your class could define __dir__ as follows::
```

Comment:
Do we need to escape the underscores?

Reply:
No, I don't think so - but you could wrap in double back-ticks if you want it formatted as an inline literal.